its/src/test/kotlin/org/sonarlint/intellij/its/tests/ConnectedIdeaTests.kt

```diff
@@ -283,6 +287,37 @@ class ConnectedIdeaTests : BaseUiTest() {
 
     }
 
+    @Tag("Suite2")
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    inner class SharedConfigTests : BaseUiTest() {
+        @BeforeAll
+        fun initProfile() {
+            ORCHESTRATOR.server.restoreProfile(FileLocation.ofClasspath("/connectedmode-json-java-issue.xml"))
+            ORCHESTRATOR.server.provisionProject(CONNECTEDMODE_JSON_PROJECT_KEY, "ConnectedMode Json")
+            ORCHESTRATOR.server.associateProjectToQualityProfile(
+                CONNECTEDMODE_JSON_PROJECT_KEY,
+                "java",
+                "SonarLint IT ConnectedMode Java Issue"
+            )
+            // Build and analyze project to raise issue
+            executeBuildWithMaven("projects/connectedmode-json/pom.xml", ORCHESTRATOR)
+            firstIssueKey = getFirstIssueKey(adminWsClient)
+        }
+
+        @Test
+        fun should_receive_shared_config() = uiTest {
+            openExistingProject("connectedmode-json")
+            enableConnectedModeFromCurrentFilePanel(CONNECTEDMODE_JSON_PROJECT_KEY, true, "Orchestrator")
+            notification("Share configuration").click()
+            shareConfiguration()
+            clearConnections()
+            goBackToWelcomeScreen()
+            openExistingProject("connectedmode-json")
+            notification("Use configuration")
```

Comment:
It could worth testing if the connection was successfully imported

Reply:
Whenever we open a project, it uses the configuration information we initially stored in the main project(under its/shared-connected-mode/.sonarlint. So even after we use export it would be using the URL with the old port which makes it not possible to test this feature since we are not connected to it and we do not have the token for it.

Reply:
You can test if you export and import in the same test - so you don't have to restart orchestrator (which would modify the port)